### ISSUE TITLE
Accessibility related fixes

### ISSUE THREAD

**Issue by waynedyck** [WSDOT/external-html-template#9]
Several of these can be fixed in the static HTML template

The external link indicator is causing issues. 
- **IMAGE MISSING TEXT ALTERNATIVE**: The graphic should be marked as decorative so screen readers can ignore it. 
- **VISIBLE LABEL & ACCESSIBLE NAME DO NOT MATCH**: The aria label is “link is external” which is read to screen readers, but doesn’t share what the actual link is – not sure how to fix this.  

**LINK MISSING TEXT ALTERNATIVE**: Get help in your language link in header needs a text alternative.

Tabbed pages are causing issues: 
- **ROLE NOT INSIDE THE REQUIRED CONTEXT**: It reads only the first item in the tabbed list. 
- **CONTAINER ELEMENT IS EMPTY**: doesn’t recognize tabs as items in the tab list.  
- **CONTENT MISSING AFTER HEADING; EMPTY HEADINGS; HIDDEN ELEMENT HAS FOCUSABLE CONTENT**: Screen readers only pick up on tab H2s- doesn’t show that there is content within each tab. 

**HEADINGS ARE NOT STRUCTURED**: Because of “how are we doing banner,” how are we doing banner should probably be marked as a H3 rather than H2.

**OVERUSE OF ITALICS**: Remove italic formatting from Project page image captions.

**TEXT NOT INCLUDED IN AN ARIA LANDMARK**:
- All perceivable text content should be included in an [ARIA landmark](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/).
